spec/graphql/analysis/ast/field_usage_spec.rb

```diff
@@ -138,6 +138,20 @@
     end
   end
 
+  describe "query with deprecated enum argument" do
+    let(:query_string) {%|
+      query {
+        fromSource(source: YAK) {
+          id
+        }
+      }
+    |}
+
+    it "keeps track of deprecated arguments" do
+      assert_equal ['Query.fromSource.source'], result[:used_deprecated_arguments]
```

Comment:
The interesting thing is `Query.fromSource.source` is the argument name, but it's the argument value `YAK` in this case that's deprecated. I wonder if there's a path we should return that is more explicit about the problem?